src/elements/pages/access-token.scss

```diff
@@ -0,0 +1,28 @@
+@use '../shared/common';
+
+#base {
+	display: flex;
+	flex-direction: column;
+	align-items: center;
+	justify-content: center;
+	height: 100%;
+
+	h2 {
+		font-size: 35px;
+		margin: 0 0 5px 0;
+		color: #ececec;
+
+		e-svg {
+			width: 1.1em;
+			height: 1.1em;
+			margin-right: 0.3em;
+			vertical-align: -0.2em;
+		}
+	}
+
+	p {
+		margin: 0 0 20px 0;
+		font-weight: 600;
+		color: #ececec;
+	}
+}
```

Comment:
![](https://media.tenor.com/fBvQV_5Lp6UAAAAC/we-dont-do-that-here-black-panther.gif)

Please use Tailwind! :)

Reply:
You can reuse styling from `rvt-user-dashboard`, so the design matches throughout the app.

Reply:
`rvt-user-dashboard` has scss too, is that just leftover code?

Reply:
https://github.com/rivet-gg/hub/pull/28/commits/e93b09f9d86bdb38ff3fabfd43b4b532f180c253